packages/core/src/app.ts

```diff
@@ -259,9 +264,7 @@ export class AppBuilder<TreeNames> {
     private app: Package,
     private adapter: AppAdapter<TreeNames>,
     private options: Required<Options>
-  ) {
-    MacrosConfig.shared().setOwnConfig(__filename, { active: true });
-  }
+  ) {}
```

Comment:
Can we remove the constructor completely?

Reply:
ya